Source/FolioReaderPage.swift

```diff
@@ -425,7 +425,11 @@ public class FolioReaderPage: UICollectionViewCell, UIWebViewDelegate, UIGesture
     // MARK: UIMenu visibility
     
     override public func canPerformAction(action: Selector, withSender sender: AnyObject?) -> Bool {
-        if UIMenuController.sharedMenuController().menuItems?.count == 0 {
+		guard readerConfig.useReaderMenuController else {
+			return false
```

Comment:
I don't think return false is the best option in this case, we will have problem while overriding probably. Also if the custom menu is disable it need to enable the default webView menu.

In my point of view the ideal is to call super here, so the default menu will opens, and when overriding you can chose if whether to show custom or not.

```swift
guard readerConfig.useReaderMenuController else {
	return super.canPerformAction(action, withSender: sender)
}
```